.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,39 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
```

Comment:
Do we need to add a separate job for lint?

Reply:
Yes, added it